jekyll/_cci2/deploy-to-heroku.adoc

```diff
@@ -57,9 +57,9 @@ jobs:
     deploy:
     docker:
         - image: <docker-image-name-tag>
-        auth:
-            username: mydockerhub-user
-            password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+            auth:
```

Comment:
Extra indentation?

Reply:
@kbatuigas those aren't extra indentations, those are pages that have tabs set instead of spaces for indentation and changing it on my settings in VS code for some reason didn't change the whole page. These whole pages need to be formatted differently for indentation.